### PR TITLE
Dev si 2nd sprint

### PR DESCRIPTION
**Pull request by JRojas999**

### CODE REVIEW

File: src/main/java/org/fundacionjala/at13/spaceinvaders/GameBoard.java [hunk 1]
```diff
@@ -43,15 +45,59 @@ public void init() {
...
+
+    /**
+     * @Override keyReleased.
+    */
+    @Override
+    public void keyReleased(final KeyEvent e) {
+        return;
+    }
+
+    /**Refresh the icon of a label where it is our spaceship.*/
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacionjala/at13/spaceinvaders/GameBoard.java:93]: The description of this method should be in 3 lines as in the other methods

File: src/main/java/org/fundacionjala/at13/spaceinvaders/GameBoard.java [hunk 2]
```diff
@@ -43,15 +45,59 @@ public void init() {
...
+    public void refresh() {
         ImageIcon iconLogo = new ImageIcon("resources/spaceship.png");
         labelArray[spaceship.getPosY()][spaceship.getPosX()].setIcon(iconLogo);
     }
 
+    /** Clean the icon of a label where it was our spaceship.*/
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacionjala/at13/spaceinvaders/GameBoard.java:99]: The description of this method should be in 3 lines as in the other methods

File: src/main/java/org/fundacionjala/at13/spaceinvaders/GameBoard.java [hunk 3]
```diff
@@ -43,15 +45,59 @@ public void init() {
...
+        addKeyListener(this);
     }
 
     /**
      * Method to initialize and show the spaceShip.
      */
     public void spaceShip() {
         spaceship = new Spaceship(SCALE_WIDTH / 2, SCALE_HEIGHT - 1, 0, SCALE_WIDTH);
+        /* ImageIcon iconLogo = new ImageIcon("resources/spaceship.png");
+        labelArray[spaceship.getPosY()][spaceship.getPosX()].setIcon(iconLogo); */
```
Comment (cristianfchq) [inline src/main/java/org/fundacionjala/at13/spaceinvaders/GameBoard.java:57]: if these two lines are not necessary you can delete them

File: src/main/java/org/fundacionjala/at13/spaceinvaders/GameBoard.java [hunk 4]
```diff
@@ -43,15 +45,59 @@ public void init() {
...
         spaceShip();
+        addKeyListener(this);
     }
 
     /**
      * Method to initialize and show the spaceShip.
      */
     public void spaceShip() {
         spaceship = new Spaceship(SCALE_WIDTH / 2, SCALE_HEIGHT - 1, 0, SCALE_WIDTH);
+        /* ImageIcon iconLogo = new ImageIcon("resources/spaceship.png");
```
Comment (jpgonzales27) [inline src/main/java/org/fundacionjala/at13/spaceinvaders/GameBoard.java:56]: Maybe you should delete these commented lines

**Review by danielaSantaCruz — APPROVED** [review]

**Review by jpgonzales27 — APPROVED** [review]

**Review by cristianfchq — APPROVED** [review]

**Pull request merged**